### PR TITLE
[refactor](ecosystem) refactor flink doris connector doc

### PR DESCRIPTION
**Pull request by JNSimba**
## Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [x] 2.0

## Languages

- [x] Chinese
- [x] English

## Docs Checklist

- [ ] Checked by AI
- [ ] Test Cases Built

### CODE REVIEW

File: common_docs_zh/ecosystem/flink-doris-connector.md [hunk 1]
```diff
@@ -28,251 +28,332 @@ under the License.
 
 # Flink Doris Connector
```
Comment (KassieZ) [inline common_docs_zh/ecosystem/flink-doris-connector.md:29]: 可删除

File: common_docs_zh/ecosystem/flink-doris-connector.md [hunk 2]
```diff
@@ -28,251 +28,332 @@ under the License.
...
-| 1.5.2             | 1.15,1.16,1.17,1.18 | 1.0+ | 8 |- |
-| 1.6.2             | 1.15,1.16,1.17,1.18,1.19 | 1.0+ | 8 |- |
-| 24.0.1            | 1.15,1.16,1.17,1.18,1.19,1.20 | 1.0+ | 8 |- |
+# 使用方式
 
-## 使用
+## 安装配置
 
-### Maven
+可在[这里](https://repo.maven.apache.org/maven2/org/apache/doris/)下载Flink Doris Connector对应版本的Jar包，将此文件复制到 `Flink` 的 `classpath` 中即可使用 `Flink-Doris-Connector` 。 `Standalone` 模式运行的 `Flink` ，将此文件放入 `lib/` 文件夹下。 `Yarn` 集群模式运行的 `Flink` ，则将此文件放入预部署包中。
```
Comment (KassieZ) [inline common_docs_zh/ecosystem/flink-doris-connector.md:54]: 下载链接已就绪。请引导至官网：https://doris.apache.org/download#doris-ecosystem

File: common_docs_zh/ecosystem/flink-doris-connector.md [hunk 3]
```diff
@@ -740,84 +625,239 @@ insert into doris_sink select id,name,bank,age from cdc_mysql_source;
     --table-conf replication_num=1
 ```
 
-## 使用 Flink CDC 更新 Key 列
+# 参数配置
```
Comment (KassieZ) [inline common_docs_zh/ecosystem/flink-doris-connector.md:628]: `## 参数配置`

File: common_docs_zh/ecosystem/flink-doris-connector.md [hunk 4]
```diff
@@ -740,84 +625,239 @@ insert into doris_sink select id,name,bank,age from cdc_mysql_source;
...
+
+## 通用配置项
```
Comment (KassieZ) [inline common_docs_zh/ecosystem/flink-doris-connector.md:630]: `### 通用配置项`，下方标题均需要调整

**Review by KassieZ — COMMENTED** [review]
中英文之间需要添加空格，可使用工具调整：

image

**Comment by KassieZ** [comment]
request review @wangtianyi2004

File: common_docs_zh/ecosystem/flink-doris-connector.md (hunk 3 above)
Comment (JNSimba) [reply common_docs_zh/ecosystem/flink-doris-connector.md:628]: This is the first-level directory, which is aligned with 【快速上手、使用方式】

File: common_docs_zh/ecosystem/flink-doris-connector.md (hunk 4 above)
Comment (JNSimba) [reply common_docs_zh/ecosystem/flink-doris-connector.md:628]: Same as above

File: common_docs_zh/ecosystem/flink-doris-connector.md (hunk 2 above)
Comment (JNSimba) [reply common_docs_zh/ecosystem/flink-doris-connector.md]: done

File: common_docs_zh/ecosystem/flink-doris-connector.md (hunk 1 above)
Comment (JNSimba) [reply common_docs_zh/ecosystem/flink-doris-connector.md]: done

**Pull request merged**